push.py

```diff
@@ -17,8 +17,8 @@
 parser.add_argument('-b', '--branch', help="log choosen preCICE branch")
 parser.add_argument('-t', '--test', help="choose system tests you want to use")
 parser.add_argument('-s', '--success', action='store_true' ,help="only upload log file")
-parser.add_argument('-o', '--os', type=str,help="Version of Ubuntu to use", choices =
-            ["1804", "1604"], default= "1604")
+parser.add_argument('-o', '--os', type=str,help="Distribution to use", choices =
```

Comment:
Is that really a distribution or the named variant of the base docker file? Do you adhere to https://github.com/precice/systemtests/issues/14?

Reply:
I read #14, but here the same argument as for `system_testing.py` applies - right now we don't test anything else, but  'Ubuntu1604' and 'Ubuntu1804' on Travis, and I beleve with @MakisH  we concluded that is enough. 
 But if we decide to test and push results from different variants  ( as in #14 ) based on the same operating system, I would change this bit.

Reply:
Well, if we don't have the infrastructure in place we will never be able to test something else, therefore we don't do it.

We should definitely be able to test on arbitrary base images, as laid out in #14. If you feel that this scheme is not suitable, please comment here.

For travis only testing Ubuntu is ok, but the test suite should test other variants as well. None of supercomputers uses Ubuntu.

Reply:
I confirm @shkodm that we agreed (between the two of us) that we need to be able on different distributions and that for Travis Ubuntu is enough. I am not sure I understand the problem here and what should be done differently.

Reply:
Allright, you are right, I will adjust that bit.